README.rst

```diff
@@ -7,3 +7,8 @@ The prime libCellML repository is located at: https://github.com/cellml/libcellm
 
 The documentation for the libCellML project is available at: http://libcellml.readthedocs.org/.
 
+Scope
+=====
+
+The purpose of libCellML is to create, manipulate, serialize, deserialize, validate and instantiate CellML models.  It is not concerned with retrieving CellML imports from external sources.
```

Comment:
Good point Jonathan. I think we should discuss this a bit more on the mailing list before adding it in. @hsorby, perhaps remove this change from this set and I'll create a new issue regarding the scope.